htdocs/salaries/card.php

```diff
@@ -709,6 +709,24 @@ function( data ) {
 		//$formquestion[] = array('type' => 'date', 'name' => 'clone_date_ech', 'label' => $langs->trans("Date"), 'value' => -1);
 		$formquestion[] = array('type' => 'date', 'name' => 'clone_date_start', 'label' => $langs->trans("DateStart"), 'value' => -1);
 		$formquestion[] = array('type' => 'date', 'name' => 'clone_date_end', 'label' => $langs->trans("DateEnd"), 'value' => -1);
+		echo "<script>		
+		window.onload = () => {			
+			$('#clone_date_start').change(() => {			
+				const date = $('#clone_date_start').val();
+				const days = parseInt(date.split('/')[1]);
```

Comment:
Instead of analysing a non structured string, you can analyse $('#clone_date_startday"), $('#clone_date_startmonth"), $('#clone_date_startyear"),   to get component of date. Analysing string will not work, because the format differs on language (and is very exotic with some languages).